src/webextension/manage/components/item-details.js

```diff
@@ -62,27 +61,42 @@ export default class ItemDetails extends React.Component {
         e.preventDefault();
         onSave(this.state);
       }}>
-        <p>
-          <Localized id="item-site">
-            <span>sITe</span>
+        <label>
+          <Localized id="item-details-title">
+            <span>tITLe</span>
```

Comment:
Personal curiosity, but why a nested `<span>` tag instead of using `<label htmlFor="xyz">tITLe</label>` and then giving an `id` attribute to the `<Input>` tags?

Reply:
IDs are global across the entire page, which is a leaky abstraction; if I started using IDs, I'd have to manage ID uniqueness across supposedly-independent components. We already have to do that with localization keys, but I hope to eliminate those issues at some point too. (We could use some sort of unique ID generator, but that's extra complexity for something I can get for free by simply avoiding IDs in the first place).

That said, I might be able to just give `<Localized>` some text.